app/assets/stylesheets/application.css

```diff
@@ -15,16 +15,25 @@
  */
 
 .navbar {
-    background-color: lightgray;
-    height: 50px;
+  background-color: lightgray;
+  height: 50px;
 }
 .navbar_logo {
-    color: black;
+  color: black;
    
 }
 .category {
-    color: black;
+  color: black;
 }
 .subcategory {
-    color: black;
+  color: black;
+}
+
+#js-search {
```

Comment:
try to use classes for styling instead of ids

Reply:
ok